app/src/main/java/ch/epfl/skysync/ui/components/Timer.kt

```diff
@@ -0,0 +1,73 @@
+package ch.epfl.skysync.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.delay
+
+@Composable
+fun Timer(modifier: Modifier) {
+  var timer by remember { mutableIntStateOf(0) }
+  var isRunning by remember { mutableStateOf(false) }
+  Box(
+      modifier = modifier.testTag("Timer"),
+      contentAlignment = androidx.compose.ui.Alignment.Center) {
+        Column(horizontalAlignment = androidx.compose.ui.Alignment.CenterHorizontally) {
+          Text(
+              modifier = Modifier.testTag("Timer Value"),
+              text = timer.formatTime(),
+              style = MaterialTheme.typography.headlineMedium)
+          if (isRunning) {
+            Button(
```

Comment:
Would've been nice to add color = lightOrange to keep the visuals of the other buttons on the screen

Reply:
It's done